### PR TITLE
docs: add qwik-image inside the Image Optimisation section

### PR DESCRIPTION
**Pull request by gioboa**
# Overview

I create the [Qwik Image](https://github.com/qwikifiers/qwik-image) component inside the qwikifiers GitHub account.
This component follow the Qwik APIs guidelines and I think this is a great plus.
This PR is to add qwik-image library inside the Image Optimisation section.

# What is it?

- [X] Docs / tests

# Checklist:

- [X] I have performed a self-review of my own code
- [X] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/docs/src/routes/docs/integrations/image-optimization/index.mdx [hunk 1]
```diff
@@ -14,27 +14,68 @@ import CodeSandbox from '../../../../components/code-sandbox/index.tsx';
 
 Image optimization is the process of reducing the number of bytes that represent an image. The smaller the image size, the faster it will load on a page. The smaller the image size, the less bandwidth it consumes. The less bandwidth it consumes, the better the experience for your users especially on mobile networks.
 
+## `qwik-image`
+
+### The goal is a component to support performant images on the web and automatic image optimization.<br><br>
```
Comment (manucorporat) [inline packages/docs/src/routes/docs/integrations/image-optimization/index.mdx:19]: This title is too long

**Review by manucorporat — CHANGES_REQUESTED** [review]

File: packages/docs/src/routes/docs/integrations/image-optimization/index.mdx (hunk 1 above)
Comment (gioboa) [inline packages/docs/src/routes/docs/integrations/image-optimization/index.mdx:19]:
```suggestion
### Performant images with automatic optimisation.<br><br>
```

Comment (gioboa) [reply packages/docs/src/routes/docs/integrations/image-optimization/index.mdx]: Changed

**Comment by manucorporat** [comment]
lets merge it!

**Comment by manucorporat** [comment]
lets merge it!

**Pull request merged**